src/ASM/Test/TestASMs2Dmx.C

```diff
@@ -0,0 +1,297 @@
+//==============================================================================
+//!
+//! \file TestASMs2Dmx.C
+//!
+//! \date Aug 25 2023
+//!
+//! \author Arne Morten Kvarving / SINTEF
+//!
+//! \brief Tests for structured 2D mixed spline FE models.
+//!
+//==============================================================================
+
+#include "ASMmxBase.h"
+#include "ASMSquare.h"
+
+#include "gtest/gtest.h"
+
+
+namespace {
+
```

Comment:
what is the effect of this namespace with no name? You refer to its members below as if the namespace was not there. Is it really needed? And if so, why?

Reply:
Right, that was what I missed, the `static` keyword was gone.
The content could heave been indented though, to make that anonymous namespace more "visible" for reader.

For the class-scoped string you however have `static constexpr const` prefix, which looks abit over-specified (but is probably with reason as well)

Reply:
i'm not used to indenting for namespaces. and yes, but i need that because only static constexpr variables can be inline initialized, if not i'd need a compile unit to instance in.